apps/studio/src/common/formUtils.ts

```diff
@@ -209,17 +209,25 @@ export const commonYupValidation = {
   password: Yup.string().required("Password is required"),
   publicationDate: Yup.date().max(new Date(), "Cannot be a future date"),
   releaseDate: (releaseDate: string | undefined) => {
-    // If releaseDate is provided, use it.
-    // Otherwise, use the minimum time for EVEARA to distribute from now.
+    // This is necessary to avoid time zone discrepancies,
+    // as Date object uses the local time zone by default.
+    const parseDateAsUTC = (dateString: string) => {
+      const [year, month, day] = dateString.split("-");
+      return new Date(
+        Date.UTC(parseInt(year), parseInt(month) - 1, parseInt(day))
+      );
+    };
```

Comment:
I don't think the `parseDateAsUtc` is needed at all. The function is doing it's job to convert to UTC but the conversion doesn't really matter in the `min` comparison below because ISO already converts the release date `Date` object to UTC and `.split("T")[0]` provides it in the same format as in the Field Value. It just matters that the Dates are formatted in the same YEAR-MONTH-DAY format at that point of comparison. 

The main issue before was that we were comparing Unix values, including hour of the day in the local area, which would not be valid. Really the big difference is now the comparison is less rigid by only using dates.

```
releaseDateFieldValue -> "2024-02-12"

new Date(minReleaseDate).toISOString().split("T")[0]  -> '2024-02-12'
```

You can just remove the `parseDateAsUTC` and change the `minReleaseDate` ternary back to `new Date(releaseDate)` and I believe this will all still work.